diff -up ksh-1.0.0/src/cmd/ksh93/data/strdata.c.1~ ksh-1.0.0/src/cmd/ksh93/data/strdata.c
--- ksh-1.0.0/src/cmd/ksh93/data/strdata.c.1~	2022-08-02 22:38:35.072534451 +0200
+++ ksh-1.0.0/src/cmd/ksh93/data/strdata.c	2022-08-02 22:43:27.499339059 +0200
@@ -17,6 +17,10 @@
 /*
  * data for string evaluator library
  */
+#define _BITS_FLOATN_COMMON_H // ksh doesn't seem to like _Float* types at all
+#define _BITS_FLOATN_H
+#define _Float64x double
+#define _Float128 long double
 
 #include	"shopt.h"
 #include	<ast_standards.h>
